src/driver/franka_plan_runner.cc

```diff
@@ -736,10 +807,10 @@ franka::JointPositions FrankaPlanRunner::JointPositionCallback(
     Eigen::VectorXd dq_abs {
         utils::v_to_e(utils::ArrayToVector(cannonical_robot_state.dq))
             .cwiseAbs()};
-    dexai::log()->warn(
-        "JointPositionCallback: plan {} overtime, "
-        "franka_t: {:.3f}, max joint err: {:.4f}, speed norm: {:.5f}",
-        plan_utime_, franka_time_, max_joint_err, dq_abs.norm());
+    // dexai::log()->warn(
+    //     "JointPositionCallback: plan {} overtime, "
+    //     "franka_t: {:.3f}, max joint err: {:.4f}, speed norm: {:.5f}",
+    //     plan_utime_, franka_time_, max_joint_err, dq_abs.norm());
```

Comment:
According to @gavincangan these printouts can affect performance by causing the callback to not be executed at the correct frequency... should we put them in a separate thread?

Reply:
We start seeing communication constraint violation if the control loop takes more than ~400 us

Reply:
Slows it down and causes comm constraint violations if this gets printed in every iteration. Observed on jello. We could make this a separate thread instead and detach it

Reply:
moved to separate thread